ste/mgr-JobPartMgr.go

```diff
@@ -670,7 +670,7 @@ func (jpm *jobPartMgr) updateJobPartProgress(status common.TransferStatus) {
 		atomic.AddUint32(&jpm.atomicTransfersSkipped, 1)
 	case common.ETransferStatus.Cancelled():
 	default:
-		panic("Unexpected status")
+		jpm.Log(pipeline.LogError, fmt.Sprintf("Unexpected status: %v", status.String()))
```

Comment:
This does avoid the panic, but is this log even useful?

Reply:
If there are other races or changes which would lead us to this scenario, we'd want to know.